.github/workflows/parachain.yml

```diff
@@ -102,6 +102,7 @@ jobs:
             --release
             --verbose 
             --workspace
+            --exclude snowbridge-query-events
```

Comment:
There are no tests in this package anyway and there is a compiler bug when testing with `cargo tarpaulin` where the linker fails.